### PR TITLE
[TS] LPD-18823 Add integration test

### PR DESCRIPTION
**Pull request by evibudai**
Hi Team,

https://liferay.atlassian.net/browse/LPD-18823

I'm sending the additional integration test to review.

Thanks!

### CODE REVIEW

**Review by G-Sa — CHANGES_REQUESTED** [review]
Hey @evibudai i noticed these, can you take a look please?!

**Review by G-Sa — APPROVED** [review]
Lgtm :tada:

File: modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java [hunk 1]
```diff
@@ -405,6 +406,52 @@ public void testPutObjectDefinition() throws Exception {
...
+
+		Assert.assertEquals(
+			accountEntryObjectDefinitionResult.getExternalReferenceCode(),
+			accountSystemObjectDefinition.getExternalReferenceCode());
+
+		ObjectRelationship objectRelationshipResult =
+			accountEntryObjectDefinitionResult.getObjectRelationships()[0];
+
+		Assert.assertNotNull(objectRelationshipResult.getObjectDefinitionId2());
+
```
Comment (carolmariaabb) [inline modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java:454]: @evibudai please simply the code to:

```
// Account entry update with null objectDefinitionId2

accountEntryObjectDefinition.setExternalReferenceCode(
	RandomTestUtil.randomString());
accountEntryObjectDefinition.setTitleObjectFieldName(
	"type");

ObjectRelationship objectRelationship =
	accountEntryObjectDefinition.getObjectRelationships()[0];

objectRelationship.setObjectDefinitionId2((Long)null);

ObjectDefinition putAccountEntryObjectDefinition =
	objectDefinitionResource.putObjectDefinition(
		accountEntryObjectDefinition.getId(),
		accountEntryObjectDefinition);

Assert.assertEquals(
	accountEntryObjectDefinition.getExternalReferenceCode(),
	putAccountEntryObjectDefinition.getExternalReferenceCode());
Assert.assertEquals(
	accountEntryObjectDefinition.getTitleObjectFieldName(),
	putAccountEntryObjectDefinition.getTitleObjectFieldName());

objectRelationship =
	putAccountEntryObjectDefinition.getObjectRelationships()[0];

Assert.assertNotNull(objectRelationship.getObjectDefinitionId2());

```

My changes:
- I reused the existing variable `accountEntryObjectDefinition`. 
- I ordered alphabetically: `externalReferenceCode` before `titleObjectFieldName`. 
- In the `Assert.assertEquals`, we must place the expected result before the actual result.

Futhermore, after we do:

```
ObjectRelationship objectRelationship =
	accountEntryObjectDefinition.getObjectRelationships()[0];

objectRelationship.setObjectDefinitionId2((Long)null);
```
we don't need to do:

```
accountEntryObjectDefinition.setObjectRelationships(
	new ObjectRelationship[]{objectRelationship});
```

the `objectRelationship` is already the same object inside the `accountEntryObjectDefinition` object.

Thanks!

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

**Comment by evibudai** [comment]
@carolmariaabb thanks for the review! I pushed the changes.

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**